dcurve/src/internal/calc/linear.ts

```diff
@@ -25,32 +33,44 @@ export const handle = (clrArgs: CLRArgs): GrantsDistribution => {
         grantAddress: contribution.grantAddress,
         contributions: [],
       };
+
+      contributionAddresses.push(contribution.address);
```

Comment:
Nit: could save a line and make intent more clear. Instead of array-> set -> array, could just do

```typescript
const contributionAddresses: Set<string> = new Set();
// ...
contributionAddresses.add(contribution.address)
// ...
const trustBonusScores = await fetchTrustBonusScore([...contributionAddresses]);